### ISSUE TITLE
Fail: Maximum content size limit

### ISSUE THREAD

**Issue by yenthesoloman** [chidiwilliams/buzz#381]
Hi there, I got an error when trying to transcribe with openai whisper

Failed (Maximum content size limit (26214400) exceeded (222592852 bytes read) {
  "error": {
    "message": "Maximum content size limit (26214400) exceeded (222592852 bytes read)",
    "type": "server_error",
    "param": null,
    "code": null
  }
} 413 {'error': {'message': 'Maximum content size limit (26214400) exceeded (222592852 bytes read)', 'type': 'server_error', 'param': None, 'code': None}} {'Date': 'Tue, 21 Mar 2023 02:27:18 GMT', 'Content-Type': 'application/json', 'Content-Length': '172', 'Connection': 'keep-alive', 'Openai-Processing-Ms': '1279', 'Openai-Version': '2020-10-01', 'Strict-Transport-Security': 'max-age=15724800; includeSubDomains', 'X-Ratelimit-Limit-Requests': '3000', 'X-Ratelimit-Remaining-Requests': '2999', 'X-Ratelimit-Reset-Requests': '20ms', 'X-Request-Id': '9445746f1ab72e34ee52518ceb0030da'})

**Comment by chidiwilliams** [comment]
Yes, I need to work on this for smaller files. Should be done this weekend or so.

**Comment by tobiasdiez** [comment]
I still experience this issue.

**Comment by pcuellar** [comment]
Same here
